### PR TITLE
Deploy Heroku

### PR DESCRIPTION
**Pull request by ronyyosef**
Online site:  https://xpense-beyond.herokuapp.com
There is an admin already created: name=admin, password=admin1234
For now, only I will deploy to Heroku from my local machine 
We will think about implementing a pipeline of deployment in the future

### CODE REVIEW

**Comment by ronyyosef** [comment]
https://xpense-beyond.herokuapp.com/house/

**Comment by shlior7** [comment]
maybe consider deploying on vercel it is easier and I used that for all my react project
it seems to have small configuration process 
https://jay-hale.medium.com/django-on-vercel-in-30-minutes-e69eed15b616

**Comment by ronyyosef** [comment]
Anyone know how to handle this Errors
[pipenv.exceptions.InstallError]:     If you prefer to avoid building psycopg2 from source, please install the PyPI
[pipenv.exceptions.InstallError]:     'psycopg2-binary' package instead.
[pipenv.exceptions.InstallError]:     
[pipenv.exceptions.InstallError]:     For further information please check the 'doc/src/install.rst' file (also at
[pipenv.exceptions.InstallError]:     <https://www.psycopg.org/docs/install.html>).
[pipenv.exceptions.InstallError]:     
[pipenv.exceptions.InstallError]:     ----------------------------------------
[pipenv.exceptions.InstallError]: WARNING: Discarding https://files.pythonhosted.org/packages/d1/1e/b450599a27b1809bccbd4e369f397cb18dc56b875778d961f9ae180b[54](https://github.com/redhat-beyond/Xpense/runs/6614811329?check_suite_focus=true#step:5:55)b7/psycopg2-2.9.3.tar.gz#sha2[56](https://github.com/redhat-beyond/Xpense/runs/6614811329?check_suite_focus=true#step:5:57)=8e841d1bf3434da985cc5ef13e6f75c8981ced601fd70cc6bf33351b91562981 (from https://pypi.org/simple/psycopg2/) (requires-python:>=3.6). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
[pipenv.exceptions.InstallError]: ERROR: Could not find a version that satisfies the requirement psycopg2==2.9.3
[pipenv.exceptions.InstallError]: ERROR: No matching distribution found for psycopg2==2.9.3
ERROR: Couldn't install package: psycopg2
 Package installation failed...

**Review by Yoavsc129 — APPROVED** [review]

**Comment by LiorKGOW** [comment]
> pipenv.exceptions.InstallError]: ERROR: Could not find a version that satisfies the requirement psycopg2==2.9.3
> [pipenv.exceptions.InstallError]: ERROR: No matching distribution found for psycopg2==2.9.3
> ERROR: Couldn't install package: psycopg2

Maybe one of these comments can help:
https://github.com/pypa/pipenv/issues/3991#issuecomment-543197653
with this adition: https://github.com/pypa/pipenv/issues/3991#issuecomment-545219494
https://github.com/pypa/pipenv/issues/3991#issuecomment-569417731
https://github.com/pypa/pipenv/issues/3991#issuecomment-564645309

**Comment by ronyyosef** [comment]
Solved using other library

**Review by LiorKGOW — APPROVED** [review]
LGTM 👍

**Review by ShellyGolden — APPROVED** [review]

File: runtime.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+python-3.10.1
```
Comment (shlior7) [inline runtime.txt:1]: didn't we configure our project as python 3.9?

Comment (ronyyosef) [reply runtime.txt:1]: Yes we did
image
Here are the supported python versions by Heroku, Do you think we should use 3.9.13?
As far as I see it works well with 3.10

Comment (shlior7) [reply runtime.txt:1]: if it works it ain't broken I suppose
but if it also works with the 3.9.13 I think it makes more sense

**Review by shlior7 — APPROVED** [review]

File: runtime.txt (hunk 1 above)
Comment (ronyyosef) [reply runtime.txt:1]: 👍

**Pull request closed without merging**